src/components/editors.tsx

```diff
@@ -40,18 +47,56 @@ export interface IProps extends EditorProps {
     onExport: () => void;
     onAddOperation: (operation: Operation) => Promise<void>;
     onOpenSwitchboardLink?: () => Promise<void>;
+    fileId: string;
 }
 
+const signOperation = async (
```

Comment:
for something with this many arguments, it might be better dx to use an object with named keys. otherwise it can be easy to mix them up